pkg/client/cache/delta_fifo_test.go

```diff
@@ -27,6 +27,24 @@ func testPop(f *DeltaFIFO) testFifoObject {
 	return f.Pop().(Deltas).Newest().Object.(testFifoObject)
 }
 
+// KeyLookupFunc adapts a raw function to be a KeyLookup.
+type KeyLookupFunc func() []string
```

Comment:
Suggest make this private.

(or put back the way it was, but I guess if no one used it, it's not a big deal)